src/app/slash_plugins/addclasschans.plugin.ts

```diff
@@ -0,0 +1,238 @@
+import {
+  CategoryChannel,
+  CommandInteraction,
+  GuildChannel,
+  Message,
+  MessageEmbed,
+  TextChannel,
+  Util,
+} from 'discord.js';
+import ms from 'ms';
+import Constants from '../../common/constants';
+import { ISlashCommand } from '../../common/slash';
+import { ClassType, IContainer } from '../../common/types';
+
+interface IChannel {
+  code: string;
+  category: string;
+  name?: string;
+}
+
+const plugin = {
+  commandName: 'addclasschans',
+  name: 'Add many classes',
+  description: 'creates a bunch of class channels',
+  defaultMemberPermissions: 'ADMINISTRATOR',
+  options: [
+    {
+      name: 'category',
+      description: 'The category of the class',
+      type: 'STRING',
+      required: true,
+      choices: [
+        { name: 'Computer Science', value: 'cs' },
```

Comment:
should this list be factored out? it's the same as in some other plugin

Reply:
Yeah this was done in one other place. I changed the code to copy that.

I dont think its worth factoring out since its a one liner.